mapper/cumulocity/c8y_translator_lib/src/serializer.rs

```diff
@@ -126,3 +126,171 @@ impl GroupedMeasurementVisitor for C8yJsonSerializer {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use assert_json_diff::*;
+    use serde_json::json;
+
+    use super::*;
+    use chrono::{offset::FixedOffset, DateTime, Local};
+    fn test_timestamp() -> DateTime<FixedOffset> {
+        let local_time_now: DateTime<Local> = Local::now();
+        local_time_now.with_timezone(local_time_now.offset())
+    }
+    #[test]
+    fn serialize_single_value_message() {
+        let mut serializer = C8yJsonSerializer::new().unwrap();
+        let timestamp = test_timestamp();
```

Comment:
use fixed timestamp in these kind of tests.

Reply:
fixed